docs/src/prototypes/popups/stickerPicker.tsx

```diff
@@ -0,0 +1,83 @@
+import * as React from 'react'
+import * as ReactDOM from 'react-dom'
+import {
+  Button,
+  Popup,
+  Menu,
+  dialogBehavior,
+  tabBehavior,
+  tabListBehavior,
+} from '@stardust-ui/react'
+import * as _ from 'lodash'
+import { arrayOfStickerImagesNames, getItemsData } from './helper'
+import GridImagePicker from './GridImagePicker/GridImagePicker'
+
+const tabListItemsContent = [
+  'Popular',
+  'Office drama',
+  'Meme',
+  'Designers',
+  'Dev',
+  'Legal',
+  'Team squatch',
+]
+
+class StickerPicker extends React.Component {
+  state = { activeMenuIndex: 0 }
+  gridPickerRef = React.createRef<GridImagePicker>()
+
+  getStickersData = () => {
+    const index = arrayOfStickerImagesNames[this.state.activeMenuIndex]
+      ? this.state.activeMenuIndex
+      : 0
+    return getItemsData(arrayOfStickerImagesNames[index], 'sticker of')
+  }
+
+  onMenuItemClick = (e, props) => {
+    this.setState({ activeMenuIndex: props.index }, () => {
+      const gridPickerElement = ReactDOM.findDOMNode(this.gridPickerRef.current) as HTMLElement
+      const input = gridPickerElement && gridPickerElement.querySelector('input')
+      input && input.focus()
```

Comment:
Sorry, but it's not cool. If we want to focus `input` we can make it in class method:

```jsx
class GridImagePicker {
  focusInput() {}
}
```

```suggestion
      this.gridPickerRef.current.focusInput()
```